web/apps/labelstudio/src/config/ApiConfig.example.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * To mock a request, you can define the endpoint inside `endpoints` with a schema that looks like this:
+ * endpointName: {                              // endpointName is what is used when calling callAPI, passed in as a string
+ *   method?: "GET",                            // defaults to GET. options are GET, POST, PATCH, DELETE
+ *   path: "/tasks/:taskID/annotations",        // actual endpoint pattern you want to mock
+ *   mock?: (url, params, fakeRequest) => any,  // function that returns the mock data wrapped into internal response object
+ *                                              //    accepts string: url, json object: params and Request: fakeRequest
```

Comment:
hmm, why do you use this approach to describe params? usual way is `name: type`, not vise versa

Reply:
i wanted to be explicit about the params, but you definitely have point - it would be a very TS approach to it